vital/types/image_container.h

```diff
@@ -97,6 +97,8 @@ typedef std::shared_ptr<image_container> image_container_sptr;
 
 
 /// List of image_container shared pointers
+// NOTE(paul.tunison): This should be deprecated in favor of
+//                     vital::image_container_set_sptr.
```

Comment:
Isn't there a macro for deprecation? I thought I saw it in some of Linus' code, but can't find it now.

Reply:
A call to ``git grep deprec`` does not show anything for me.

Reply:
The deprecated macro is generated along with the export macro. I would have added it here, but I don't know where to pout it in the line. (Also don't know if you can deprecate a typedef.)